x/evm/keeper/msg_server.go

```diff
@@ -34,11 +34,13 @@ func (k *Keeper) EthereumTx(goCtx context.Context, msg *types.MsgEthereumTx) (*t
 	if tx.To() == nil {
 		labels = []metrics.Label{
 			telemetry.NewLabel("execution", "create"),
+			telemetry.NewLabel("from", sender),
```

Comment:
let's move this to L33

Reply:
Just a reminder that this also goes for line 43